### PR TITLE
Disable flacky test PyQgsPostgresRasterProvider

### PR DESCRIPTION
**Pull request by elpaso**
Dear Travis: you won.

### CODE REVIEW

**Comment by nyalldawson** [comment]
Shall we call it on the azure 32 bit builds too? There's absolutely 0 clues to go on to fix these...

**Comment by elpaso** [comment]
> Shall we call it on the azure 32 bit builds too? There's absolutely 0 clues to go on to fix these...

is the test failing on azure too?

**Comment by nyalldawson** [comment]
> is the test failing on azure too?

No, there's just been a bunch of completely random failures on the 32 bit builds for ~ 1 week. No apparent cause to them, and no discernible pattern.

**Comment by elpaso** [comment]
No idea if this could help on Azure, feel free to disable.

File: .ci/travis/linux/scripts/test_flaky.txt [hunk 1]
```diff
@@ -8,3 +8,6 @@ qgis_ziplayertest
 # the docker base image is upgraded
 PyQgsMssqlProvider
 
+# Flacky
```
Comment (m-kuhn) [inline .ci/travis/linux/scripts/test_flaky.txt:11]:
```suggestion
# Flaky
```

**Review by m-kuhn — APPROVED** [review]
+1 to disable

**Pull request merged**